testsuite/features/reposync/srv_sync_products.feature

```diff
@@ -118,25 +118,33 @@ Feature: Synchronize products in the products page of the Setup Wizard
     And I enter "SUSE Manager Proxy 4.3" as the filtered product description
     And I select "SUSE Manager Proxy 4.3 x86_64" as a product
     Then I should see the "SUSE Manager Proxy 4.3 x86_64" selected
+    When I open the sub-list of the product "SUSE Manager Proxy 4.3 x86_64"
+    And I open the sub-list of the product "Basesystem Module 15 SP4 x86_64"
+    And I select "Containers Module 15 SP4 x86_64" as a product
+    Then I should see the "Containers Module 15 SP4 x86_64" selected
     When I click the Add Product button
     And I wait until I see "Selected channels/products were scheduled successfully for syncing." text
     And I wait until I see "SUSE Manager Proxy 4.3 x86_64" product has been added
     And I wait until all synchronized channels for "suma-proxy-43" have finished
 
 @proxy
 @susemanager
-  Scenario: Add SUSE Manager Retail Branch Server
+  Scenario: Add SUSE Manager Retail Branch Server 4.3
```

Comment:
oh, thanks! :smile_cat: 

but this scenario (and the previous one) will create a problem with SUSE Manager 5.0 :disappointed: .

should we change all 4.3 to 5.0 (only in uyuni branch) ? Ping @nodeg .

Reply:
I have no strong opinion about that. We can diverge with this between SUMA and Uyuni.

Reply:
It was rather beteen suma 4.3 and suma 5.0. But I guess I already answered myself, we'll just switch when 5.0 is out.

Reply:
I also have no strong opinion about that.